exports.func_createEmployee = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
}

exports.func_getEmployee = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
}

exports.func_getEmployeeById = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
}

exports.func_updateEmployee = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
}

exports.func_deleteEmployee = async (req, res, next) => {
    try {
        
    } catch (error) {
        next(error);
    }
}